bootstrap.php

```diff
@@ -1019,8 +1021,8 @@ function ($app) {
     }
 );
 
-$app['logger.amqp.event_bus_forwarder'] = $app->share(
-    function () {
+$app['logger.amqp.event_bus_forwarder'] = $app::share(
```

Comment:
Why do we call `::share()` statically on an instance?

Reply:
Because PHPStorm added a warning that it should be called statically and it was obfuscating other warnings inside the closure.

![Screenshot 2020-10-26 at 11 22 09](https://user-images.githubusercontent.com/959026/97161045-897db580-177d-11eb-84c6-a290c745febe.png)

It says that on every `$app->share` btw:
![Screenshot 2020-10-26 at 11 22 55](https://user-images.githubusercontent.com/959026/97161137-aadea180-177d-11eb-8028-9b86f8f130a6.png)

Might be the plugin @LucWollants recommended.

But anyway, it's a static function and it's valid syntax to call it like that, even on an instance.

Reply:
I also noticed this "issue" with the various assert methods from PHPUnit.

Reply:
I disabled that rule because phpunit expects to call non-statically even though it's still a static method.